core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -61,10 +61,25 @@ angular.module('oppia').directive('imageUploader', [
           }
 
           var ONE_MB_IN_BYTES = 1048576;
-          if (file.size > ONE_MB_IN_BYTES) {
-            var currentSize = (file.size / ONE_MB_IN_BYTES).toFixed(1) + ' MB';
-            return 'The maximum allowed file size is 1 MB' +
+          var HUNDRED_KB_IN_BYTES = 102400;
+          var ALLOWED_SIZE_IN_BYTES = null;
+          var currentSize = null;
+          var errorMessage = null;
+          if (file.type.match('image.svg')) {
```

Comment:
Weird. How do you know it's image.svg? Maybe add a reference -- it's weird that it differs from the format of image/png. I would have expected image/svg+xml or similar, so just wanted to understand how you tested this.